esmvalcore/cmor/tables/custom/CMOR_co2s.dat

```diff
@@ -0,0 +1,20 @@
+SOURCE: CMIP5
+!============
+variable_entry:    co2s
+!============
+modeling_realm:    atmos
+!----------------------------------
+! Variable attributes:
+!----------------------------------
+standard_name:     mole_fraction_of_carbon_dioxide_in_air
+units:             mol mol-1
```

Comment:
Is there a reason for not using the same units as the parent variable (`1.e-6`)?

Reply:
I used the one from the CMIP6 table:

https://github.com/ESMValGroup/ESMValCore/blob/735f4160da4d2ae113dd87f93f25cb38d943c656/esmvalcore/cmor/tables/cmip6/Tables/CMIP6_Amon.json#L305-L321

Reply:
Ok. I was looking into CMIP5.